op-program/verify/cmd/goerli.go

```diff
@@ -127,30 +132,44 @@ func Run(l1RpcUrl string, l1RpcKind string, l2RpcUrl string, l2OracleAddr common
 		"--l2.blocknumber", l2BlockNumber.String(),
 	}
 	argsStr := strings.Join(args, " ")
+	// args.txt is used by run-goerli-verify job for offline verification in CI
 	if err := os.WriteFile(filepath.Join(dataDir, "args.txt"), []byte(argsStr), 0644); err != nil {
 		fmt.Printf("Could not write args: %v", err)
 		os.Exit(1)
 	}
 	fmt.Printf("Configuration: %s\n", argsStr)
+
+	logger := oplog.DefaultCLIConfig()
+	logger.Level = log.LvlDebug
+	rollupCfg, err := rollup.LoadOPStackRollupConfig(chainconfig.OPGoerliChainConfig.ChainID.Uint64())
+	if err != nil {
+		return fmt.Errorf("failed to load rollup config: %w", err)
+	}
+	offlineCfg := &config.Config{
+		Rollup:             rollupCfg,
+		DataDir:            dataDir,
+		L2ChainConfig:      chainconfig.OPGoerliChainConfig,
+		L2Head:             l2Head,
+		L2OutputRoot:       agreedOutput.OutputRoot,
+		L2Claim:            l2Claim,
+		L2ClaimBlockNumber: l2BlockNumber.Uint64(),
+		L1Head:             l1Head,
+	}
+	onlineCfg := offlineCfg
```

Comment:
Isn't this copying the pointer and so `onlineCfg` and `offlineCfg` are actually the same struct still? I'd have thought we'd want no `&` in the declaration for offlineCfg and then just create a pointer to it when passing it to Main further down.

Reply:
Yeah this ends up being the same struct. It should be a copy. Good catch.

Reply:
I'm a REAL go programmer!!! 🤣